_episodes/03-ncbi-sra.md

```diff
@@ -13,7 +13,7 @@ keypoints:
 
 In our experiments we're usually generating our own genomic data, but many types of analyses use reference data or you may want to use it to compare your results or annotate your data with publicly available data. You may also want to do a full project or set of analyses using publicly available data. This data is a great, and essential, resource for genomic data analysis.
 
-There are many repositories for public data. Some model organisms or fields have specific databases, and there are ones for particular types of data. Two of the most comprehensive are the [National Center for Biotechnology Information  (NCBI)](https://www.ncbi.nlm.nih.gov) and [European Nucleotide Archive (EMBL-EBI)](https://www.ebi.ac.uk/ena). In this lesson we're working with the NCBI database, but the general process is the same for any database.
+There are many repositories for public data. Some model organisms or fields have specific databases, and there are ones for particular types of data. Two of the most comprehensive are provided by the [National Center for Biotechnology Information (NCBI)](https://www.ncbi.nlm.nih.gov) and the [European Bioinformatics Institute (EMBL-EBI)](https://www.ebi.ac.uk/). In this lesson we're working with the NCBI database, but the general process is the same for any database.
 
```

Comment:
To both acknowledge the institute and point to the resources, how about "the [European Bioinformatics Institute (EMBL-EBI)](https://www.ebi.ac.uk/) through the [European Nucleotide Archive](https://www.ebi.ac.uk/ena)."

Reply:
Sounds great to me. @tobyhodges  if you can make the change, we should be able to merge. Thanks!

Reply:
:+1:

Reply:
Thanks @tobyhodges!